crates/primitives/src/uopool.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(clap::ValueEnum, Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Mode {
+    Auto,
```

Comment:
`Auto` seems ambiguous to me. Any better words?

Reply:
I was thinking about "Standard". Do you have any other suggestion?